This patch allows the user to use
Vim keys to cycle through images.
Automatic GIF playing is on by default
--- a/config.def.h	2021-02-02 12:46:36.820376736 +0100
+++ b/config.def.h	2021-02-02 12:47:19.760792856 +0100
@@ -103,9 +103,11 @@
 	{ 0,            XK_n,             i_navigate,           +1 },
 	{ 0,            XK_n,             i_scroll_to_edge,     DIR_LEFT | DIR_UP },
 	{ 0,            XK_space,         i_navigate,           +1 },
+	{ 0,            XK_l,             i_navigate,           +1 },
 	{ 0,            XK_p,             i_navigate,           -1 },
 	{ 0,            XK_p,             i_scroll_to_edge,     DIR_LEFT | DIR_UP },
 	{ 0,            XK_BackSpace,     i_navigate,           -1 },
+	{ 0,            XK_h,             i_navigate,           -1 },
 	{ 0,            XK_bracketright,  i_navigate,           +10 },
 	{ 0,            XK_bracketleft,   i_navigate,           -10 },
 	{ ControlMask,  XK_6,             i_alternate,          None },

--- options.c	2021-02-02 12:54:51.555171023 +0100
+++ options.c	2021-02-02 12:55:04.795299327 +0100
@@ -56,7 +56,7 @@
 
 	_options.scalemode = SCALE_DOWN;
 	_options.zoom = 1.0;
-	_options.animate = false;
+	_options.animate = true;
 	_options.gamma = 0;
 	_options.slideshow = 0;
 	_options.framerate = 0;
